src/App.js

```diff
@@ -33,6 +34,7 @@ const ProfilePage = lazy(() => import('./pages/ProfilePage'));
 const EditProfilePage = lazy(() => import('./pages/EditProfilePage'));
 const CartPage = lazy(() => import('./pages/CartPage'));
 const AboutPage = lazy(() => import('./pages/AboutPage'));
+const CreateReviewPage = lazy(() => import('./pages/CreateReviewPage'));
```

Comment:
Why you decided to use lazy() here?

Reply:
I use lazy to have less files loaded initially, so that the app can have better performance. Once this page is not going to be accessed so often i think is good to use lazy to only bring its code when requeted.